lib/galaxy/tool_util/deps/container_classes.py

```diff
@@ -16,6 +16,7 @@
 )
 from uuid import uuid4
 
+from packaging.version import Version
```

Comment:
I think you want to use our vendored copy (galaxy.tool_util.parse_version) here that will not fail on invalid versions

Reply:
Will do. But I would hope that Galaxy versions always have been and always will be valid :)

Reply:
yes, but I wouldn't count on that at e.g. https://github.com/galaxyproject/galaxy/pull/16492/files#diff-7071c96be315e9633878a25245ae69ccd1267efefef3ee41f87d0512d6c48dceR367

Reply:
ah, ok, if these would already not load then that's ok